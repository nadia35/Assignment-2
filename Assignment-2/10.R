my_values <- c(1,2,3,4)
retrieveEvenIndices <- function(my_values){
  
  even_indices <- my_values[seq(2, length((my_values)), 2)]
  
  return(even_indices)
}

