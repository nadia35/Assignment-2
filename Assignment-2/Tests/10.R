source("10.R")

print(retrieveEvenIndices(c(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)))
print(retrieveEvenIndices(c(9, 8, 7, 6, 5, 4, 3, 2, 1, 0)))
print(retrieveEvenIndices(c(0, 0, 0, 0, 0, 0, 0, 1)))
print(retrieveEvenIndices(c(98.183, 800.0, 561.238)))