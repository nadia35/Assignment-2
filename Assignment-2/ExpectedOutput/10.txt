[1] 1 3 5 7 9
[1] 8 6 4 2 0
[1] 0 0 0 1
[1] 800
